cmd/server/main.go

```diff
@@ -92,7 +94,18 @@ func main() {
 		db = database.NewPostgresStore(*psqlDsn)
 	}
 	// Start the endpoint
-	s, err := server.NewRpcEndPointServer(logger, version, *listenAddress, *relayUrl, *proxyUrl, *proxyTimeoutSeconds, key, *redisUrl, db)
+	s, err := server.NewRpcEndPointServer(server.Configuration{
```

Comment:
**explanation:**

instead of passing a train of arguments to the `NewRpcEndPointServer` (which would render it error-prone due to arguments being misplaced) let's use `struct` that will have all of them properly named.